pkg/provider/flipt/provider.go

```diff
@@ -144,9 +122,21 @@ func (p Provider) getFlag(ctx context.Context, flag string) (*flipt.Flag, of.Pro
 	return f, of.ProviderResolutionDetail{}, nil
 }
 
+func splitNamespaceAndFlag(src string) (string, string) {
+	ns, flag, found := strings.Cut(src, "/")
+	if found {
+		return ns, flag
+	}
+
+	return "default", src
+}
+
 // BooleanEvaluation returns a boolean flag.
 func (p Provider) BooleanEvaluation(ctx context.Context, flag string, defaultValue bool, evalCtx of.FlattenedContext) of.BoolResolutionDetail {
-	f, res, err := p.getFlag(ctx, flag)
+	namespace, flagKey := splitNamespaceAndFlag(flag)
+
+	// TODO: we have to check if the flag is enabled here until https://github.com/flipt-io/flipt/issues/1060 is resolved
```

Comment:
Just saying this out loud. Don't need to address it in this PR.

I believe we can now switch to using `Reason` on the `EvaluationResponse` to figure out the resolution details more precisely without having to call GetFlag.
That is a future refactor though.

Reply:
@GeorgeMac I did that in this PR. Thanks for the suggestion! We can remove the extra network hop 👍